pkg/sql/importer/import_planning.go

```diff
@@ -814,6 +814,10 @@ func importPlanHook(
 				return err
 			}
 
+			if len(found.LDRJobIDs) > 0 {
+				return errors.Newf("cannot run an import on table %s which is apart of a Logical Data Replication stream", table)
```

Comment:
nit: "apart" means separated. (but even "a part" is kinda strange since it isn't _part_ of the stream? maybe just say "in use by"?)